config/sync/system.logging.yml

```diff
@@ -1,3 +1,3 @@
-error_level: hide
+error_level: all
```

Comment:
not sure we'd want to show all errors on prod? (it's been a while since I did D8, so not sure if there's a different solution to the dev/prod config sync issues now)